src/frontend/components/SøknadsSteg/Dokumentasjon/filopplaster/useFilopplaster.tsx

```diff
@@ -96,8 +96,8 @@ export const useFilopplaster = (
                                     tidspunkt: dagensDatoStreng,
                                 });
                             })
-                            .catch(_error => {
-                                const badRequestCode = badRequestCodeFraError(_error);
+                            .catch(error => {
```

Comment:
Du kan fjerne den i selve badRequestCodeFraError-funksjonen også 😊

Reply:
Ah, nice catch. Var visst litt for rask! 😅